locale/en/knowledge/intermediate/how-to-log.md

```diff
@@ -53,37 +59,45 @@ The advantage here is that the behavior of our logging mechanisms can now be mod
 
 Here is an example of setting up a `winston` logger.  This example includes most of the transports one could ever possibly want - please note that most use cases will only warrant a few of these.
 
-      var winston = require('winston');
-
-      require('winston-riak').Riak;
-      require('winston-mongo').Mongo;
-      require('winston-couchdb').Couchdb;
-
-      var logger = new (winston.Logger)({
-        transports: [
-          new winston.transports.Console(),
-          new winston.transports.File({ filename: 'path/to/all-logs.log' }),
-          new winston.transports.Couchdb({ 'host': 'localhost', 'db': 'logs' }),
-          new winston.transports.Riak({ bucket: 'logs' }),
-          new winston.transports.MongoDB({ db: 'db', level: 'info'})
-        ],
-        exceptionHandlers: [
-          new winston.transports.File({ filename: 'path/to/exceptions.log' })
-        ]
-      });
+```javascript
+var winston = require('winston');
+
+require('winston-riak').Riak;
+require('winston-mongo').Mongo;
+require('winston-couchdb').Couchdb;
+
+var logger = new (winston.Logger)({
+  transports: [
+    new winston.transports.Console(),
+    new winston.transports.File({ filename: 'path/to/all-logs.log' }),
+    new winston.transports.Couchdb({ 'host': 'localhost', 'db': 'logs' }),
+    new winston.transports.Riak({ bucket: 'logs' }),
+    new winston.transports.MongoDB({ db: 'db', level: 'info'})
+  ],
+  exceptionHandlers: [
+    new winston.transports.File({ filename: 'path/to/exceptions.log' })
+  ]
+});
+```
 
 Here, we have instantiated a new `winston` logger, and provided a number of logging transports.  Winston has built-in support for configurable logging levels, and provides alias methods for each configured logging level.  For example, `winston.warn(x)` is an alias for `winston.log('warn', x)`.  Thus, the following:
 
-      logger.warn('Hull Breach Detected on Deck 7!');
+```javascript
+logger.warn('Hull Breach Detected on Deck 7!');
+```
 
 Would output to the screen:
 
-      warn: Hull Breach Detected on Deck 7!
+```
+warn: Hull Breach Detected on Deck 7!
+```
 
 Because of the file transport we set up, winston also logged the warning to 'somefile.log'.  After the `logger.warn` call we just used, the log file, `somefile.log`, would contain the following output:
 
-      $ cat somefile.log
-      {'level':'warn','message':'Hull Breach Detected on Deck 7!'}
+```
```

Comment:
This (in all languages) could be `bash`

Reply:
Maybe just leave it as is.

Reply:
Yeah, please don't use <code>```bash</code> for this! Use that for bash scripts only. It does confusing highlighting for command-line shell stuff. Use <code>```console</code> for that instead if you really want to specify a language/formatter/highlighter.

Reply:
@Trott `console` is not a valid prism language thus this will fail.

Reply:
Yeah, we figured out that "console" chokes out metalsmith, unless some of the dependency updates done recently fix that

Reply:
See https://github.com/nodejs/nodejs.org/pull/2557#issuecomment-529292779

Reply:
Ah, OK, yeah, in that case, yes, specifying nothing is 👍.  Sorry I'm having difficulty keeping up.

Reply:
Nope, this is how metalsmith-prism works. If we want to change this, we need to fork the project.

Reply:
which has been archived BTW